platform/viewer/src/lib/DICOMSR/dataExchange.js

```diff
@@ -0,0 +1,33 @@
+import OHIF from '@ohif/core';
+import { retrieveMeasurementFromSR, stowSRFromMeasurements } from './handleSR';
+import { getLatestSRSeries } from './srUtils';
+
+export const retrieveMeasurements = () => {
+  console.info('[DICOMSR] retrieveMeasurements');
+
+  const latestSeries = getLatestSRSeries();
+
+  if (!latestSeries) return Promise.resolve({});
+
+  return retrieveMeasurementFromSR(latestSeries);
+};
+
+export const storeMeasurements = (measurementData, server) => {
+  console.info('[DICOMSR] storeMeasurements');
+
+  if (!server || server.type !== 'dicomWeb') {
+    return Promise.resolve({});
```

Comment:
We should throw an error here instead

Reply:
You are right @swederik 
I've added an error here and the code is now preventing showing the save button in case the server is not dicomWeb type.